packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -19,25 +19,15 @@ export interface PluginConfig extends Partial<PluginWorkerConfig> {
 
 type Defined<T> = Exclude<T, undefined>;
 
-export type AssetsOnlyConfig = Unstable_Config & {
-	assets: Defined<Unstable_Config['assets']>;
+export type AssetsOnlyConfig = WorkersConfigurations.SanitizedWorkerConfig & {
+	assets: Defined<WorkersConfigurations.SanitizedWorkerConfig['assets']>;
 };
 
-export type WorkerConfig = Unstable_Config & {
-	name: Defined<Unstable_Config['name']>;
-	main: Defined<Unstable_Config['main']>;
+export type WorkerConfig = WorkersConfigurations.SanitizedWorkerConfig & {
+	name: Defined<WorkersConfigurations.SanitizedWorkerConfig['name']>;
+	main: Defined<WorkersConfigurations.SanitizedWorkerConfig['main']>;
```

Comment:
NIT: perhaps move these to `workers-configs.ts`?

Reply:
I think the line here is quite blurry, because these `WorkerConfig`s are relate/used by to the plugin itself 🤔